portfolio/src/main/webapp/script.js

```diff
@@ -16,12 +16,51 @@
  * Cycles through greetings in different languages for front page. 
  */
 const greetings =['Hi, nice to meet you!', '¡Hola, encantado de conocerte!', '你好，很高興見到你', 'Salut! Enchanté!'];
-var textElement = document.getElementById('changeText');
+let textElement = document.getElementById('changeText');
 textElement.innerHTML = greetings[0];
-var counter = 1;
-var intervalLength = setInterval(change, 2000);
+let counter = 1;
+setInterval(change, 2000);
 document.getElementById('commentBtn').addEventListener('click', getComments);
 
+class Landmark {
+  constructor(title, lat, lng, description) {
+    this.title = title;
+    this.lat = lat;
+    this.lng = lng;
+    this.description = description;     
+  }
+}
+
+function initMap() {
+    const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422403, lng: -122.088073}, zoom: 5});
+    
+    let landmarks = [new Landmark('Home', 37.3230, -122.0322, 'Where I currently live, and spent my high school years. \
+    Home to Apple HQ, the best boba, and my closest friends :)'), new Landmark('UMass Amherst', 42.3868, -72.5301, 'Snowy, \
+    remote, but cozy; where I spent my first 1.5 years of college.'), new Landmark('Georgia Tech', 33.7756, -84.3963, 'Where I \
+    currently attend college!'), new Landmark('Home, Away From Home', 19.4564, 72.7925, 'Where my parents grew up; the comfort \
+    of my extended family and the cities unique culture always brings me back'), new Landmark('An Extraordinary Layover',
+    51.5074, -0.1278, 'Spent an ambitious 8 hours searching for all the famous landmarks, check out pictures in misc!') ]
+
+    for (let landmark of landmarks) {
+        addLandmark(map, landmark)
+    }
+};
+
+function addLandmark(map, Landmark) {
+  const marker = new google.maps.Marker(
+      {position: {lat: Landmark.lat, lng: Landmark.lng}, map: map, title: Landmark.title});
+  const infoWindow = new google.maps.InfoWindow({content: Landmark.description});
+
+  marker.addListener('click', () => {
```

Comment:
Consider closing previous markers? If one is open when I click another one, they're both open which is a bit cluttered when I zoom out. But it's subjective - whatever you want the experience to be.